src/io/flutter/logging/FlutterLogColorPage.java

```diff
@@ -0,0 +1,101 @@
+                                          "<finest>02-02 18:52:57.132: FINEST/FlutterIsCool: log message</finest>\n" +
+                                          "<finer>02-02 18:52:57.132: FINER/FlutterIsCool: log message</finer>\n" +
+                                          "<fine>02-02 18:52:57.132: FINE/FlutterIsCool: log message</fine>\n" +
+                                          "<config>02-02 18:52:57.132: CONFIG/FlutterIsCool: log message</config>\n" +
+                                          "<info>02-02 18:52:57.132: INFO/FlutterIsCool: log message</info>\n" +
+                                          "<warning>02-02 18:52:57.132: WARNING/FlutterIsCool: log message</warning>\n" +
+                                          "<severe>02-02 18:52:57.132: SEVERE/FlutterIsCool: log message</severe>\n" +
+                                          "<shout>02-02 18:52:57.132: SHOUT/FlutterIsCool: log message</shout>";
+
+  static {
+    ADDITIONAL_HIGHLIGHT_DESCRIPTORS.put("none", FlutterLogConstants.NONE_OUTPUT_KEY);
+    ADDITIONAL_HIGHLIGHT_DESCRIPTORS.put("finest", FlutterLogConstants.FINEST_OUTPUT_KEY);
+    ADDITIONAL_HIGHLIGHT_DESCRIPTORS.put("finer", FlutterLogConstants.FINER_OUTPUT_KEY);
+    ADDITIONAL_HIGHLIGHT_DESCRIPTORS.put("fine", FlutterLogConstants.FINE_OUTPUT_KEY);
+    ADDITIONAL_HIGHLIGHT_DESCRIPTORS.put("config", FlutterLogConstants.CONFIG_OUTPUT_KEY);
+    ADDITIONAL_HIGHLIGHT_DESCRIPTORS.put("info", FlutterLogConstants.INFO_OUTPUT_KEY);
+    ADDITIONAL_HIGHLIGHT_DESCRIPTORS.put("warning", FlutterLogConstants.WARNING_OUTPUT_KEY);
+    ADDITIONAL_HIGHLIGHT_DESCRIPTORS.put("severe", FlutterLogConstants.SEVERE_OUTPUT_KEY);
+    ADDITIONAL_HIGHLIGHT_DESCRIPTORS.put("shout", FlutterLogConstants.SHOUT_OUTPUT_KEY);
+  }
+
+  @NotNull
+  private static final AttributesDescriptor[] ATTRIBUTES_DESCRIPTORS =
+    new AttributesDescriptor[]{
+      new AttributesDescriptor(FlutterBundle.message("none.level.title"), FlutterLogConstants.NONE_OUTPUT_KEY),
+      new AttributesDescriptor(FlutterBundle.message("finest.level.title"), FlutterLogConstants.FINEST_OUTPUT_KEY),
+      new AttributesDescriptor(FlutterBundle.message("finer.level.title"), FlutterLogConstants.FINER_OUTPUT_KEY),
+      new AttributesDescriptor(FlutterBundle.message("fine.level.title"), FlutterLogConstants.FINE_OUTPUT_KEY),
+      new AttributesDescriptor(FlutterBundle.message("config.level.title"), FlutterLogConstants.CONFIG_OUTPUT_KEY),
+      new AttributesDescriptor(FlutterBundle.message("info.level.title"), FlutterLogConstants.INFO_OUTPUT_KEY),
+      new AttributesDescriptor(FlutterBundle.message("warning.level.title"), FlutterLogConstants.WARNING_OUTPUT_KEY),
+      new AttributesDescriptor(FlutterBundle.message("severe.level.title"), FlutterLogConstants.SEVERE_OUTPUT_KEY),
+      new AttributesDescriptor(FlutterBundle.message("shout.level.title"), FlutterLogConstants.SHOUT_OUTPUT_KEY)};
+
```

Comment:
No big deal but I notice that these are showing in alphabetical sorted order in the view when sorted like you have here would be better.  I wonder if there's any way we could do that?

For what it's worth, it looks like the Logcat view has the same (I think bad) behavior:

![image](https://user-images.githubusercontent.com/67586/41658376-07b74bce-744b-11e8-95ed-0fe1b48fa819.png)

Reply:
I will add a TODO. (as I don't know how to fix that problem - it may be automatically sort inside intellij)

Reply:
Perfect.  Thanks!